src/network/bmproto.py

```diff
@@ -334,7 +334,7 @@ def bm_command_object(self):
         self.object = BMObject(nonce, expiresTime, objectType, version, streamNumber, self.payload, self.payloadOffset)
 
         if len(self.payload) - self.payloadOffset > BMProto.maxObjectPayloadSize:
-            logger.info('The payload length of this object is too large (%s bytes). Ignoring it.' % len(self.payload) - self.payloadOffset)
+            logger.info('The payload length of this object is too large (%d bytes). Ignoring it.' % len(self.payload) - self.payloadOffset)
```

Comment:
doesn't it have to be in brackets, like `(len(self.payload) - self.payloadOffset))`?

Reply:
Indeed, updated.